.ci/311-DEV.yaml

```diff
@@ -3,19 +3,21 @@ channels:
   - conda-forge
 dependencies:
   - python=3.11
-  - geopandas
```

Comment:
I like to keep the packages installed from main also in here because this ensures that their dependencies get installed from conda. If you keep them only below, shapely and other stuff will come from PyPI.

Reply:
Ah ha.... This is a very good point.

Reply:
d18503c